package/gluon-mesh-batman-adv/src/Makefile

```diff
@@ -31,5 +31,8 @@ endif
 CFLAGS += $(LIBBATADV_CFLAGS)
 LDLIBS += $(LIBBATADV_LDLIBS)
 
-respondd.so: respondd.c
-	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -shared -fPIC -D_GNU_SOURCE -o $@ $^ $(LDLIBS) -lgluonutil -luci
+
+SOURCES = respondd.c respondd-nodeinfo.c respondd-statistics.c respondd-neighbours.c
```

Comment:
where doesrespondd.c come from?

Reply:
From the package's `src` directory, just like before?

Reply:
yeah, I was tricked by the github gui. nevermind.